src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DBFilterBase.java

```diff
@@ -17,6 +21,37 @@
 
 /** Base for the DB filters / conditions that we want to use with queries */
 public abstract class DBFilterBase implements Supplier<BuiltCondition> {
+  /** Filter condition element path. */
+  private final String path;
+
+  protected DBFilterBase(String path) {
+    this.path = path;
+  }
+
+  /**
+   * Get JsonNode for the representing filter condition value.
+   *
+   * @param nodeFactory
+   * @return
+   */
+  abstract JsonNode asJson(JsonNodeFactory nodeFactory);
+
+  /**
+   * Returns filter condition element path.
+   *
+   * @return
+   */
+  protected String getPath() {
+    return path;
+  }
+
+  /**
+   * Returns `true` if the filter condition can be added to upsert row
```

Comment:
Should perhaps say "should be added" instead of "can be added" -- since this is what happens, right?
(or needs to be added)